src/Codacy/Coverage/Command/Clover.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace Codacy\Coverage\Command;
+
+use Symfony\Component\Console\Command\Command as ConsoleCommand;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use Codacy\Coverage\Parser\CloverParser;
+use Codacy\Coverage\Util\JsonProducer;
+use Codacy\Coverage\Util\GitClient;
+use Codacy\Coverage\Util\CodacyApiClient;
+
+/**
+ * Class Clover
+ *
+ */
+class Clover extends ConsoleCommand {
+
+    protected function configure()
+    {
+        $this
+            ->setName("clover")
+            ->setDescription("Send coverage results from clover.xml format")
+            ->addArgument(
+                "path_to_coverage_results",
+                InputArgument::OPTIONAL,
+                "Path where coverage results saved"
+            )
+            ->addOption(
```

Comment:
I need. 

My PHPUnit has very long time execution. 
So I had already prepared clover.xml. In this situation I need:
* checkout for commit
* run PHPUnit
* run codacycoverage

Second reason - CI using. For working, I need go to build dir, and exec codacy in build dir. Sometimes make this seem like workaround. So if I have report in clover.xml and has commit, I simply want to send results, without any `cd` or `chroot`. 

It's option, not a argument.